src/main/java/com/staxrt/tutorial/services/RentService.java

```diff
@@ -20,9 +18,8 @@
 public class RentService {
     @Autowired
     JdbcTemplate jdbcTemplate;
-
     private final UserRepository userRepository;
-
+    private final RentRepository rentRepository;
     private final CarRepository carRepository;
     public List<Rent> getAllRents(){
         String sql = "SELECT * FROM rents";
```

Comment:
move to class private final static variable, * is bad pratice in case table grows in the future